lib/background_worker.rb

```diff
@@ -1,15 +1,19 @@
-module BackgroundWorker
-  VERSION = '0.0.1'
-
-  require 'background_worker/base'
-  require 'background_worker/persistent_state'
+require 'background_worker/version'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.